pkg/controller.v1/pytorch/controller.go

```diff
@@ -437,7 +437,9 @@ func (pc *PyTorchController) reconcilePyTorchJobs(job *pyv1.PyTorchJob) error {
 
 	if pc.Config.EnableGangScheduling {
 		minAvailableReplicas := getTotalReplicas(job)
-		_, err := pc.SyncPodGroup(job, minAvailableReplicas)
+		priorityClassName:=getPriorityClassName(job)
+		//_, err := pc.SyncPodGroup(job, minAvailableReplicas)
+		_, err := pc.SyncPodGroupTest(job, minAvailableReplicas,priorityClassName)
```

Comment:
Why should we use SyncPodGroupTest instead of SyncPodGroup

Reply:
I add these codes in SyncPodGroupTest：
`Spec: v1alpha1.PodGroupSpec{
			MinMember: minAvailable.IntVal,
			PriorityClassName: priorityClassName,
		},
`

the name of this function is inappropriate，it is used to test my idea.

Reply:
@gaocegege Do I detele my  SyncPodGroupTest function and move the code my wrote into  original SyncPodGroup function?

Reply:
I think so.

Reply:
I got it.